aiorospy/src/aiorospy/action.py

```diff
@@ -28,3 +35,120 @@ def __init__(self, ns, action_spec, execute, loop=None):
     def _execute_cb(self):
         goal = self.accept_new_goal()
         future = asyncio.run_coroutine_threadsafe(self._execute(goal), self._loop)
+
+
+class AsyncGoalHandle:
+
+    def __init__(self, loop=None):
```

Comment:
loop looks non-optional here.  Did you mean to have a:
`loop = asyncio.get_event_loop() if loop is None else loop` ?

Reply:
Yeah, there's no `self._loop` in this class so I left it out